lang_java/parsing/test_parsing_java.ml

```diff
@@ -127,6 +132,28 @@ let test_parse_json_babelfish file =
   let str = Ocaml.string_of_v v in
   pr str
 
+let test_visitor_new file = 
```

Comment:
Please add comment on what this does!

Reply:
Same as test_visitor, except it also prints out the tokens (it didn't previously, not sure if this was a feature or a bug). Borrowed code from the PHP test_visitor function.